docs/repositories-configure/integrations/bitbucket-integration.md

```diff
@@ -52,6 +52,20 @@ Adds comments on the lines of the pull request where Codacy finds new issues. Cl
 
 ![Pull request comment on Bitbucket](images/bitbucket-integration-pr-comment.png)
 
+### AI-enhanced comments
+
+{%
+    include-markdown "../../assets/includes/paid.md"
+    start="<!--paid-start-->"
+    end="<!--paid-end-->"
+%}
```

Comment:
Since the feature is still in "private beta", I would consider that this status sort of "overrides" the paid feature status.

You should definitely confirm this with Product, but there's a chance that we can wait until there's an official communication around the pricing of the feature before we mark it as paid. For now, [including the action item](https://codacy.slite.com/app/docs/NotkMtv3mJCayS#0c71073c) to request activating the feature could be enough. :shrug:

Reply:
Good point. [Following up on Slack](https://codacy.slack.com/archives/C05158QM4E9/p1688128764922429).

Reply:
We decided not to mark the feature a paid. I added a new admonition to:
* clarify that the feature is under development and available on request
* point the user to contact support if they want access